starknet-rpc-test/get_block_transaction_count.rs

```diff
@@ -0,0 +1,88 @@
+extern crate starknet_rpc_test;
+
+use starknet_accounts::SingleOwnerAccount;
+use starknet_core::chain_id;
+use starknet_core::types::{BlockId, BlockTag};
+use starknet_ff::FieldElement;
+use starknet_providers::Provider;
+use starknet_rpc_test::constants::{ARGENT_CONTRACT_ADDRESS, MINT_AMOUNT, SIGNER_PRIVATE};
+use starknet_rpc_test::utils::transfer_tokens;
+use starknet_rpc_test::{BlockCreation, ExecutionStrategy, MadaraClient};
+use starknet_signers::{LocalWallet, SigningKey};
+use tokio::sync::OnceCell;
+
+static MADARA_CLIENT: OnceCell<MadaraClient> = OnceCell::const_new();
+
+async fn get_madara_client() -> &'static MadaraClient {
+    MADARA_CLIENT.get_or_init(|| async { MadaraClient::new(ExecutionStrategy::Native).await }).await
+}
+
+#[tokio::test]
+async fn work_ok_with_empty_block() -> Result<(), anyhow::Error> {
+    let madara = get_madara_client().await;
+    let rpc = madara.get_starknet_client();
+
+    assert_eq!(rpc.get_block_transaction_count(BlockId::Tag(BlockTag::Latest)).await?, 0);
```

Comment:
This won't work if a previous test using the same static Madara client sent some transactions but did not seal the block.

Reply:
Or is creating a block the only way to send transactions?

Reply:
Still, you have no certainty that previous tests didn't send any tx and that the latest block does not contain any.
Except if you rely on the fact that this test should be executed first, on a untouched new node. Something we  shouldn't assume

Reply:
> Or is creating a block the only way to send transactions?

Since we are running the tests sequentially and that one madara instance is shared *only* throughout its test file then I think it's fair to assume this, no ? Otherwise I can just create a new empty block before this test, fair enough